pkg/ccl/streamingccl/streamingest/stream_ingestion_job_test.go

```diff
@@ -251,7 +251,7 @@ func TestTenantStreamingFailback(t *testing.T) {
 	sqlA.Exec(t, "ALTER VIRTUAL CLUSTER f STOP SERVICE")
 	waitUntilTenantServerStopped(t, serverA.SystemLayer(), "f")
 	t.Logf("starting replication g->f")
-	sqlA.Exec(t, fmt.Sprintf("SELECT crdb_internal.unsafe_revert_tenant_to_timestamp('f', %s)", ts1))
+	sqlA.Exec(t, "ALTER VIRTUAL CLUSTER f RESET DATA TO SYSTEM TIME ($1::decimal)", ts1)
```

Comment:
Any thoughts on how we might make managing the timestamp a bit easier?

Reply:
fwiw, the casts aren't required if you run it by hand on the cli -- with an immediate value inline; they're only required if you use placeholders, since the `expr` otherwise can be any type, so it can't figure out whether the placeholder is int or string I think?

Reply:
I guess if I switch from d_expr to a_expr I at least don't need the `()` around it, but I don't think you can get away from the casts.

Reply:
Maybe `CREATE VIRTUAL CLUSTER g FROM REPLICATION OF f ON $1 WITH RESET AND RESUME TIMESTAMP = '%s'` ?

Reply:
maybe we can just discover the ts that the source-nee-dest has available and then reset to that and then resume from that?